migration/sql-files/041-work-item-link-revisions.sql

```diff
@@ -0,0 +1,22 @@
+-- create a revision table for work item links, using the some columns + identity of the user and timestamp of the operation
+CREATE TABLE work_item_link_revisions (
+    id uuid primary key DEFAULT uuid_generate_v4() NOT NULL,
+    revision_time timestamp with time zone default current_timestamp,
+    revision_type int NOT NULL,
+    modifier_id uuid NOT NULL,
+    work_item_link_id uuid NOT NULL,
+    work_item_link_version int NOT NULL,
+    work_item_link_source_id bigint NOT NULL,
+    work_item_link_target_id bigint NOT NULL,
+    work_item_link_type_id uuid NOT NULL
+);
+
+CREATE INDEX work_item_link_revisions_work_itemlinks_idx ON work_item_link_revisions USING BTREE (work_item_link_id);
+
+ALTER TABLE work_item_link_revisions
+    ADD CONSTRAINT work_item_link_revisions_identity_fk FOREIGN KEY (modifier_id) REFERENCES identities(id);
```

Comment:
I would just use the name of this table and the name of this column to be honest.